### PR TITLE
chore(oauth): Add client config for Mozilla IAM

### PR DESCRIPTION
**Pull request by rfk**
Per https://github.com/mozilla/fxa-oauth-server/issues/515

### CODE REVIEW

**Comment by vladikoff** [comment]
@rfk make sure to save the non hashed secret!

File: roles/oauth/templates/config.json.j2 [hunk 1]
```diff
@@ -337,6 +337,15 @@
       "redirectUri": "",
       "trusted": true,
       "canGrant": true
+    },
+    {
+      "id": "1a118eae25503e99",
+      "hashedSecret": "19a305e07dc4fdc502321f82745fb0c0de80d31cfa26976ad1117f898cbd1c81",
+      "name": "Mozilla IAM",
+      "imageUri": "",
+      "redirectUri": "http://localhost/",
```
Comment (vladikoff) [inline roles/oauth/templates/config.json.j2]: what should this be?

**Comment by rfk** [comment]
> make sure to save the non hashed secret!

This matches the secret they're already using on stable-dev, so hopefully they saved it :-)

File: roles/oauth/templates/config.json.j2 (hunk 1 above)
Comment (rfk) [reply roles/oauth/templates/config.json.j2]: @gdestuynder do you want to start using e.g. https://auth-dev.mozilla.auth0.com/login for integration with our dev environments?

**Comment by gdestuynder** [comment]
Hi,
We have the secret saved on dev. Hmm, not sure if we should share hashes publicly though, even thus they're hashes ;-)
Also yes redirect URIs for dev is `https://auth-dev.mozilla.auth0.com/login` - once whitelisted and set as redirect we can test it on the dev env without faking the redirect locally, it would be cool to have for the POC :)

thanks!

**Comment by rfk** [comment]
>  redirect URIs for dev is https://auth-dev.mozilla.auth0.com/login

I've updated the PR to use this instead of localhost.

**Pull request merged**